test/dockerfiles/install-debian.sh

```diff
@@ -2,20 +2,19 @@
 
 set -eu -o pipefail
 
-echo "Running Infra connector install on Ubuntu"
+echo "Running Infra connector install on $DESTINATION_NAME"
 
 PACKAGE_PATH=/work/dist/infra_0.0.0_amd64.deb
-DESTINATION_ADDR=127.0.0.1:8220
 INFRA_SERVER_URL=test-server-1
-INFRA_ACCESS_KEY=dest000001.ubuntuubuntuubuntuubuntu
 
 
 # step=install-package
+apt-get update && apt-get install --no-install-recommends -y \
+    openssh-server procps
```

Comment:
Debian install failed because `procps` wasn't installed (at least on `slim`, in practice it's probably always installed). Our package depends on it, so normally a user would be fine to just run `apt --fix-broken install`, but we don't want to handle the error here, so I moved it before we `dpkg -i` our package.